src/main/java/org/ambraproject/wombat/service/XmlUtil.java

```diff
@@ -90,6 +98,11 @@ public static String extractElement(String xml, String tagName) throws IOExcepti
     return createXmlString(extractElement(createXmlDocument(xmlSource), tagName));
   }
 
+  public static String extractElement(String xml, String tagName, String attrName, String attrValue) throws IOException {
```

Comment:
nice utility!

Reply:
was surpised this hadn't already been implemented, but suspect this utility implements just enough. so, added a bit more.